modules/core/src/main/java/com/google/refine/model/changes/ColumnRenameChange.java

```diff
@@ -89,18 +89,176 @@ static public Change load(LineNumberReader reader, Pool pool) throws Exception {
+            if (equal == -1) {
+                // This might be a continuation line from corrupted legacy data
+                continue;
+            }
+
+            String field = line.substring(0, equal);
             String value = line.substring(equal + 1);
 
             if ("oldColumnName".equals(field)) {
-                oldColumnName = value;
+                oldColumnName = processFieldValue(value, reader);
             } else if ("newColumnName".equals(field)) {
-                newColumnName = value;
+                newColumnName = processFieldValue(value, reader);
             }
         }
 
-        ColumnRenameChange change = new ColumnRenameChange(oldColumnName, newColumnName);
+        return new ColumnRenameChange(oldColumnName, newColumnName);
+    }
+
+    /**
+     * Process a field value, handling both new escaped format and legacy format. For legacy format, handle potential
+     * corruption from newline characters.
+     */
+    private static String processFieldValue(String value, LineNumberReader reader) throws IOException {
+        if (containsEscapeSequences(value)) {
+            return unescape(value);
+        } else {
+            // Legacy format - might be corrupted if original value contained newlines
+            return handleLegacyValue(value, reader);
+        }
+    }
+
+    /**
+     * Handle potentially corrupted legacy values by reading additional lines until we find a proper field or end
+     * marker.
+     */
+    private static String handleLegacyValue(String initialValue, LineNumberReader reader) throws IOException {
+        StringBuilder result = new StringBuilder(initialValue);
+
+        // Mark current position in case we need to reset
+        reader.mark(4096);
+        String nextLine;
+
+        while ((nextLine = reader.readLine()) != null) {
+            // Check if this line looks like a proper field or end marker
+            if ("/ec/".equals(nextLine) ||
+                    (nextLine.contains("=") &&
+                            (nextLine.startsWith("oldColumnName=") || nextLine.startsWith("newColumnName=")))) {
+                // This is a proper line, reset reader to before this line
+                reader.reset();
+                break;
+            } else {
+                // This might be a continuation of the corrupted value
+                result.append("\n").append(nextLine);
+                reader.mark(4096);
+            }
+        }
+
+        return result.toString();
+    }
+
+    /**
+     * Check if a value contains escape sequences that indicate new format
+     */
+    private static boolean containsEscapeSequences(String value) {
+        return value.contains("\\n")
+                || value.contains("\\r")
+                || value.contains("\\t")
+                || value.contains("\\b")
+                || value.contains("\\f")
+                || value.matches(".*\\\\u[0-9a-fA-F]{4}.*")
+                || value.contains("\\\\");
```

Comment:
If we end up using this, it can be simplified into a single regex. (but wait until we decide whether we need it at all)